### PR TITLE
fix: add isMobile condition

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[defect] 手機版點擊後會導致無法滑動](https://app.asana.com/0/1203699264568808/1207465762212942/f)

# Notice

# Dependency

### CODE REVIEW

File: packages/react-article-components/src/components/img-with-placeholder/index.js [hunk 1]
```diff
@@ -202,10 +208,21 @@ export default class Img extends React.PureComponent {
...
   componentWillUnmount() {
     this._isMounted = false
+    window.removeEventListener('resize', this.handleWindowResize)
+  }
+
+  _handleWindowResize = () => {
+    const windowWidth = window.innerWidth
+
+    this.setState({
+      isMobile: windowWidth <= 767,
```
Comment (Aylie-Chou) [inline packages/react-article-components/src/components/img-with-placeholder/index.js:224]: Could we make mobile width variable or use screen width defined in `@twreporter/core` package

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
as comment ~

File: packages/react-article-components/src/components/img-with-placeholder/index.js (hunk 1 above)
Comment (liruchen32) [reply packages/react-article-components/src/components/img-with-placeholder/index.js]: fixed

File: packages/core/src/utils/media-query.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { css } from 'styled-components'
 
-const defaultScreen = {
+export const defaultScreen = {
```
Comment (Aylie-Chou) [inline packages/core/src/utils/media-query.js:3]: Could we add `Object.freeze` for the exported const 🙏
And please help to check the naming style ([ref](https://www.notion.so/twreporter/JavaScript-Style-Guide-ca71792d06714d54af5e9ffad850bf3f?pvs=4))

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Mostly looking good !
Just need to check the exported const style & regulation 🙏

File: packages/core/src/utils/media-query.js (hunk 2 above)
Comment (liruchen32) [reply packages/core/src/utils/media-query.js]: fixed

File: packages/react-article-components/src/components/img-with-placeholder/index.js [hunk 3]
```diff
@@ -202,10 +209,21 @@ export default class Img extends React.PureComponent {
...
   componentWillUnmount() {
     this._isMounted = false
+    window.removeEventListener('resize', this.handleWindowResize)
+  }
+
+  _handleWindowResize = () => {
+    const windowWidth = window.innerWidth
+
+    this.setState({
+      isMobile: windowWidth < DEFAULT_SCREEN.tablet,
```
Comment (Aylie-Chou) [inline packages/react-article-components/src/components/img-with-placeholder/index.js:225]: DEFAULT_SCREEN.tablet`.minWidth` ?

Comment (liruchen32) [reply packages/react-article-components/src/components/img-with-placeholder/index.js]: fixed

**Review by Aylie-Chou — APPROVED** [review]
ＬＧＴＭ

**Pull request merged**